iamidentityv1/iam_identity_v1.go

```diff
@@ -43,7 +43,7 @@ type IamIdentityV1 struct {
 }
 
 // DefaultServiceURL is the default URL to make service requests to.
-const DefaultServiceURL = "https://iam.cloud.ibm.com"
+const DefaultServiceURL = "https://iam.test.cloud.ibm.com"
```

Comment:
```suggestion
const DefaultServiceURL = "https://iam.cloud.ibm.com"
```
It looks like you used the test-branch version of the API definition to generate the SDK code. You really should use the production-branch version instead.

Shortcut: note that if you are sure the exact same changes are being made to the production version of the API definition (i.e. the test and production branches differ only in the endpoint URL string, then you could safely hand-edit this constant to reflect the production endpoint URL (e.g. iam.cloud.ibm.com).

Reply:
oh yeaaa that's correct, its the same version we have in prod, so i edited out the url :) all looking good so far